### PR TITLE
added nodejs week1 homework

### PR DESCRIPTION
**Pull request by shobanamg**

### CODE REVIEW

File: nodejs/week1/avg.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const args = process.argv.slice(2);
+const numbers = args
+  .map((arg) => parseFloat(arg))
+  .filter((n) => !isNaN(n) && n !== "");
```
Comment (YurijPaharev) [inline nodejs/week1/avg.js]: Good check, but I think that checking for !isNaN is enough, as empty string would be considered as not a number either way

File: nodejs/week1/avg.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const args = process.argv.slice(2);
+const numbers = args
+  .map((arg) => parseFloat(arg))
```
Comment (YurijPaharev) [inline nodejs/week1/avg.js]: If you want to, you can simplify this block, by passing reference of this function, like `.map(parseFloat)`

**Review by YurijPaharev — APPROVED** [review]
Nice script!

File: nodejs/week1/avg.js (hunk 1 above)
Comment (shobanamg) [reply nodejs/week1/avg.js]: Silly me, true

File: nodejs/week1/avg.js (hunk 2 above)
Comment (shobanamg) [reply nodejs/week1/avg.js]: Thanks, simplified

**Pull request merged**